pkg/test/case_test.go

```diff
@@ -3,14 +3,15 @@ package test
 import (
 	"testing"
 
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	harness "github.com/kudobuilder/kuttl/pkg/apis/testharness/v1beta1"
+	testutils "github.com/kudobuilder/kuttl/pkg/test/utils"
+
```

Comment:
this isn't formatted according to our standard... I'm surprise the linters don't catch it.. but new lines sometimes evade the linters

there should be 3 groups:  core, 3rd party, this project ordered in each group